pyuntl/untldoc.py

```diff
@@ -695,3 +695,9 @@ def untl_to_hash_dict(untl_elements, meaningfulMeta=True):
         ]
     untl_tuple = untl_dict_to_tuple(untl_dict)
     return {k: generate_hash(v) for k, v in untl_tuple.items()}
+
+
+def hash_dict_to_hash_version(untl_elements):
```

Comment:
I would change the name of this to be more explanatory of the purpose of the function and because I read the name it has now as saying the argument passed in would be the hash dict (based on how other functions are named here). I suggest something like `get_record_version` or `generate_record_version`.